.gitignore

```diff
@@ -14,3 +14,7 @@ __pycache__/
 
 # Editors
 .idea/
+*.iml
+
+# Local databases.
+*.sqlite3
```

Comment:
What creates these? Are they created throughout the repository, or only at the root?

Reply:
*.iml is PyCharm/IntelliJ, and .sqlite3 is from my experiments with adding a Django example.  They are both root-only. It was convenient to .gitignore them now, as I kept accidentally adding them to changes.

Reply:
If they're root only, then you can add a `/` on the front to prevent them from being applied elsewhere.

Reply:
This PR uploaded the actual sqlite binary file.

Reply:
Ooops!

Re the `/` prefix, we still want to ignore these anywhere they might appear, they just happen to be only created in the root in practice.